### PR TITLE
RHDEVDOCS-5484-builds-redirect: redirecting to the builds standalone …

### PR DESCRIPTION
**Pull request by shipsing**
Purpose: To redirect users to the Builds standalone page

version(s): main only

Additional info: On the docs.openshift.com site, we would be having this kind of structure:
image

The existing Builds content would still be there, so no redirects for the existing Builds content.  We only need to navigate users to the builds standalone page when they look out for the Builds using Shipwright documentation. To have this structure, i created this PR: https://github.com/openshift/openshift-docs/pull/65361 and this is the preview: https://65361--docspreview.netlify.app/openshift-enterprise/latest/cicd/. 

**Context**:
On the docs.openshift.com site, we would be having this kind of Builds content structure as we now have 2 flavors for Builds:
* One is Builds using Shipwright with all the new content created from scratch in a separate standalone repo build-docs. If you expand Builds using Shipwright, we just have one file Overview of Builds under it. This file would act as a redirect point (like what we did with pipelines and gitops). Clicking this file would redirect a user to the [standalone page](https://docs.openshift.com/builds/1.0/about/overview-openshift-builds.html).
* The other is Builds using BuildConfig with old content that is going to be maintained in the docs.openshift.com site till the release of OCP 4.15. With OCP 4.15, plans are to also move the old content to the standalone repo build-docs. But for the OCP 4.14 release, the content would still be there on the docs site as is.

### CODE REVIEW

**Comment by shipsing** [comment]
@mramendi Could you please review this PR? Thanks

**Review by gabriel-rh — APPROVED** [review]
/LGTM

For redirects, you should notify Ben Scott in advance.

Also, it is worth testing all redirect changes with
https://htaccess.madewithlove.com/?share=984c5075-4e60-4474-b1f1-ea6486f64ae8

It will help pick up any typos...

**Comment by xenolinux** [comment]
LGTM

I am new on the peer review squad. Please wait until @abhatt-rh gives my review the go-ahead

**Comment by abhatt-rh** [comment]
> LGTM
> 
> I am new on the peer review squad. Please wait until @abhatt-rh gives my review the go-ahead

Redirecting to @bscott-rh for this one. Thanks, Ben!

**Comment by bscott-rh** [comment]
Redirects LGTM, just added one minor change. Thank you

File: .s2i/httpd-cfg/01-commercial.conf [hunk 1]
```diff
@@ -230,7 +230,16 @@ AddType text/vtt                            vtt
     # redirect any links to existing ROSA/Dedicated embedded content to standalone equivalent
     RewriteRule ^(rosa|dedicated)/serverless/?(.*)$ /serverless/1.30/$2  [L,R=302]
 
+    # redirect builds latest to 1.0
+    RewriteRule ^builds/?$ /builds/latest [R=302]
+    RewriteRule ^builds/latest/?(.*)$ /builds/1\.0/$1 [NE,R=302]
```
[the remark this answers is not in the document]
Comment (shipsing) [reply .s2i/httpd-cfg/01-commercial.conf]: done, Thanks

**Pull request merged**